lambda-loader/src/predictions.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+PREDICTIONS_URL = 'https://api.twitch.tv/helix/predictions'
+STREAMS_URL = 'https://api.twitch.tv/helix/streams'
+LII_TWITCH_ID = '73626243'
```

Comment:
this should be in the .env file probably

Reply:
I couldn't figure out a good way to transfer github secrets over to the environment in the lambda function so I just manually added my client_id and access token for twitch there. I will try to add these to the environment once I do figure out how to do this.